projects/ePBS-framework.md

```diff
@@ -0,0 +1,95 @@
+# ePBS Viable Paths Forward - Formal Analyses and Assessment Framework
+
+## Motivation
+*What problem is your project is solving? Why is it important and what area of the protocol will be affected?*
+
+Enshrined Proposer-Builder Separation (ePBS) is part of Ethereum's path toward better control over, if not mitigation of, toxic MEV capture as well as enabling fully stateless validators. Toxic or "bad" MEV is a major contributor of increased L1 transaction fees and a source of unfair transacting such as searchers front-running other users. Although the full extent of tactical and strategic MEV on Ethereum is not fully known, its effects are increasingly being discovered and studied.
+
+Since the [origination](https://notes.ethereum.org/@vbuterin/pbs_censorship_resistance) of the PBS problem was laid out by Vitalik in 2021, there have been a few live proposals for making a healthy transition to ePBS plausible though none selected yet as the chosen path forward.
+
+## Project description
+*What is your proposed solution?*
+
+This project dissects the original PBS problem whilst reviewing in detail the most viable and popular live proposals, which are:
+- [Why enshrine Proposer-Builder Separation? A viable path to ePBS](https://ethresear.ch/t/why-enshrine-proposer-builder-separation-a-viable-path-to-epbs/15710) by Mike Neuder and Justin Drake with [MEV burn—a simple design](https://ethresear.ch/t/mev-burn-a-simple-design/15590) which is an add-on by Justin Drake
+- [Unbundling PBS: Towards protocol-enforced proposer commitments (PEPC)](https://ethresear.ch/t/unbundling-pbs-towards-protocol-enforced-proposer-commitments-pepc/13879?u=barnabe) by Barnabé Monnot
+- [SUAVE](https://writings.flashbots.net/mevm-suave-centauri-and-beyond) (and [here](https://writings.flashbots.net/the-future-of-mev-is-suave/)) by Flashbots
+
+In dissecting the orignal PBS problem we also review the following:
+- Challenges with fair transaction sequencing, the ultimate root-cause solution to MEV extraction
+- [Forward-inclusion](https://notes.ethereum.org/@fradamt/forward-inclusion-lists) / [Proposer Broadcast lists](https://notes.ethereum.org/@fradamt/H1f3PlB5Y) `crlist`
+- [Parallelisation of block construction](https://notes.ethereum.org/@vbuterin/pbs_censorship_resistance), one of the two possible solutions proposed in Vitalik's original post
+
+## Specification
+*How will you implement your solutions? Give details and more technical information on the project.*
+
+The following deliverables should ship:
```

Comment:
I would be more specific and technical here, especially about the specs and framework part. What tools will be used, what to parametrize or model.. Of course it can be added along the way as you figure it out but I would start with more technical specs. Summary of current proposals as an outcome is nice to have but doesn't seem to include any technical analysis.